app/controllers/idv/hybrid_mobile/document_capture_controller.rb

```diff
@@ -57,8 +57,7 @@ def analytics_arguments
           irs_reproofing: irs_reproofing?,
         }.merge(
           ab_test_analytics_buckets,
-          phone_with_camera,
-        )
+        ).merge(phone_with_camera)
```

Comment:
```suggestion
          phone_with_camera, 
        )
```

Reply:
Agreed that that makes sense, but when I tried this it caused tests to fail. I'm not sure why.

Reply:
hmmm ...that's odd 🤔